### ISSUE TITLE
Disable CRD loading for restricted kubeconfig

### ISSUE THREAD

**Issue by mhoshi-vm** [vmware-tanzu/octant#2688]
**Describe the problem/challenge you have**
Related to #141.
Currently the first landing overview page, requests to full access to CRD installed on the cluster.
However limited access to CRDs.
It would be ideal if there is a flag to disable CRD loading or better is allowing users to which CRDs to load or exclude

**Describe the solution you'd like**
It would be ideal if there is a flag to disable CRD loading or better is allowing users to which CRDs to load or exclude

**Environment:**

- Octant version (use `octant version`):  0.21.0
- Kubernetes version (use `kubectl version`): 1.20.7
- OS (macOS 10.15, Windows 10, Ubuntu 19.10 etc): Ubuntu 18.04

**Comment by wwitzel3** [comment]
I agree, allowing users to explicitly provide their namespaces and resource list is something we have thought about and will be doing, but I don't have a timeline for that yet.

The current idea is that in heavily restricted environments there could two options. First, if a user has access to SelfSubjectAccessReview, Octant would walk the API and collect a list of the resources that allowed the List/Watch set of verbs.

Second, a user could provide an explicit list of GVKs to Octant using some basic config file or format. This file could be used by Octant so know which resources and namespaces to watch and would be allowed to be updated while Octant was running so that tooling could easily populate it with added/removed access.

**Comment by jpreese** [comment]
@mhoshi-vm are you using Octant in a cluster where users have access to only a single namespace? I'm getting this CRD problem as well, but Octant does not work _at all_ in our environment when permissions are reduced.

Users with full access are able to use Octant without issue.

**Comment by myminseok** [comment]
@wwitzel3 Hi, we have the same issues. 
and tried the first option you told. but the same error with forbidden to other resource like "customeresourcedefinitions", "issuers" etc even if given role of "SelfSubjectAccessReview". 
can you please elaborate more of the second option you told above? like the template or sample to test?
thank you.

**Comment by mhoshi-vm** [comment]
> @mhoshi-vm are you using Octant in a cluster where users have access to only a single namespace? I'm getting this CRD problem as well, but Octant does not work _at all_ in our environment when permissions are reduced.
> 
> Users with full access are able to use Octant without issue.

@jpreese 
Yes, so when only having a access to a single namespace my kubconfig does not work _at all_ also
But I am guessing that is because of another issue #2687 .
This ticket is for CRD specifically.
